docs/content/misc/tasks.md

```diff
@@ -251,7 +251,6 @@ The indexing service can also run real-time tasks. These tasks effectively trans
       "maxRowsInMemory": 500000,
       "intermediatePersistPeriod": "PT10m",
       "windowPeriod": "PT10m",
-      "basePersistDirectory": "\/tmp\/realtime\/basePersist",
```

Comment:
I think it was to have druid be in control of where the persistent data went, so that there was no interference between tasks.   @pjain1, you communicated with Eric about this, maybe you can clarify.